mozci/task.py

```diff
@@ -44,12 +44,17 @@ def is_no_groups_suite(label):
 bad_group_warned = False
 
 
-def is_bad_group(task_id, group):
+def is_bad_group(task_id, task_label, group):
     global bad_group_warned
 
-    bad_group = (
-        os.path.isabs(group) or group.startswith("file://") or group.startswith("Z:")
-    )
+    bad_group = group.startswith("file://") or group.startswith("Z:")
+
+    # web-platform-tests manifests are currently absolute.
```

Comment:
It'd be great if we could have web-platform-tests in the same style as other manifests (full relative path in the mozilla-central repository) to avoid special conditions here and in bugbug.